extensions/debuggee/src/org/exist/debuggee/ScriptRunner.java

```diff
@@ -69,7 +69,7 @@ public void run() {
 		try {
 			final Database db = BrokerPool.getInstance();
 			
-            BrokerPool.registerStatusObserver(this);
+//            BrokerPool.registerStatusObserver(this);
```

Comment:
@adamretter Is it not seen from my defense of this code? I did code it and I didn't say that do not support it anymore.